### ISSUE TITLE
 Failed to parse NetParameter file

### ISSUE THREAD

**Issue by tatabackstreet** [facebook/C3D#352]
I0109 16:47:49.800031  4062 caffe.cpp:275] Use GPU with device ID 0
I0109 16:47:49.843997  4062 caffe.cpp:279] GPU device name: GeForce GTX 1080 Ti
[libprotobuf ERROR google/protobuf/text_format.cc:274] Error parsing text-format caffe.NetParameter: 15:1: Expected identifier.
F0109 16:47:50.207891  4062 upgrade_proto.cpp:88] Check failed: ReadProtoFromTextFile(param_file, param) Failed to parse NetParameter file: /home/rxhlj/wh/C3D/C3D-v1.1/C3D/C3D-v1.1/examples/c3d_ucf101_finetuning/train_resnet18_r2.prototxt
*** Check failure stack trace: ***
    @     0x7ff6c380b5cd  google::LogMessage::Fail()
    @     0x7ff6c380d433  google::LogMessage::SendToLog()
    @     0x7ff6c380b15b  google::LogMessage::Flush()
    @     0x7ff6c380de1e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7ff6c3e470d1  caffe::ReadNetParamsFromTextFileOrDie()
    @     0x7ff6c3fbf732  caffe::Net<>::Net()
    @           0x409584  test()
    @           0x4075a8  main
    @     0x7ff6c1fae830  __libc_start_main
    @           0x407e79  _start
    @              (nil)  (unknown)
testing_ucf101.sh: line 1:  4062 Aborted                 (core dumped) GLOG_logtostderr=1 ./build/tools/caffe.bin test --model=/home/rxhlj/wh/C3D/C3D-v1.1/C3D/C3D-v1.1/examples/c3d_ucf101_finetuning/train_resnet18_r2.prototxt --weights=/home/rxhlj/wh/C3D/C3D-v1.1/C3D/C3D-v1.1/examples/c3d_ucf101_finetuning/c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel --stage='test-on-val' --iterations=2092 --gpu=0

I never change the prototxt file,i do not know how to solve this question,thank you.

**Comment by dutran** [comment]
It is running fine on my server: see log below.

I0129 21:39:09.648654 3538281 net.cpp:198] scale2a_branch1 needs backward computation.
I0129 21:39:09.648658 3538281 net.cpp:198] bn2a_branch1 needs backward computation.
I0129 21:39:09.648667 3538281 net.cpp:198] res2a_branch1 needs backward computation.
I0129 21:39:09.648671 3538281 net.cpp:198] conv1_conv1_relu_0_split needs backward computation.
I0129 21:39:09.648677 3538281 net.cpp:198] conv1_relu needs backward computation.
I0129 21:39:09.648682 3538281 net.cpp:198] scale_conv1 needs backward computation.
I0129 21:39:09.648686 3538281 net.cpp:198] bn_conv1 needs backward computation.
I0129 21:39:09.648690 3538281 net.cpp:198] conv1 needs backward computation.
I0129 21:39:09.648696 3538281 net.cpp:200] label_data_1_split does not need backward computation.
I0129 21:39:09.648702 3538281 net.cpp:200] data does not need backward computation.
I0129 21:39:09.648705 3538281 net.cpp:242] This network produces output accuracy/top-1
I0129 21:39:09.648711 3538281 net.cpp:242] This network produces output accuracy/top-5
I0129 21:39:09.648715 3538281 net.cpp:242] This network produces output loss
I0129 21:39:09.648773 3538281 net.cpp:255] Network initialization done.
I0129 21:39:09.762640 3538281 upgrade_proto.cpp:77] Attempting to upgrade batch norm layers using deprecated params: c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel
I0129 21:39:09.762696 3538281 upgrade_proto.cpp:80] Successfully upgraded batch norm layers using deprecated params.
I0129 21:39:09.793354 3538281 caffe.cpp:290] Running for 2092 iterations.
I0129 21:39:10.478597 3538281 caffe.cpp:313] Batch 0, accuracy/top-1 = 0.9
I0129 21:39:10.478636 3538281 caffe.cpp:313] Batch 0, accuracy/top-5 = 1
I0129 21:39:10.478641 3538281 caffe.cpp:313] Batch 0, loss = 0.345526
I0129 21:39:11.089409 3538281 caffe.cpp:313] Batch 1, accuracy/top-1 = 0.95
I0129 21:39:11.089437 3538281 caffe.cpp:313] Batch 1, accuracy/top-5 = 1
I0129 21:39:11.089453 3538281 caffe.cpp:313] Batch 1, loss = 0.274122
I0129 21:39:11.700619 3538281 caffe.cpp:313] Batch 2, accuracy/top-1 = 0.6
I0129 21:39:11.700647 3538281 caffe.cpp:313] Batch 2, accuracy/top-5 = 0.95
I0129 21:39:11.700652 3538281 caffe.cpp:313] Batch 2, loss = 1.16772
I0129 21:39:12.311959 3538281 caffe.cpp:313] Batch 3, accuracy/top-1 = 0.85
I0129 21:39:12.311988 3538281 caffe.cpp:313] Batch 3, accuracy/top-5 = 0.95
I0129 21:39:12.311993 3538281 caffe.cpp:313] Batch 3, loss = 0.661719
I0129 21:39:12.922775 3538281 caffe.cpp:313] Batch 4, accuracy/top-1 = 0.8
I0129 21:39:12.922809 3538281 caffe.cpp:313] Batch 4, accuracy/top-5 = 0.9
I0129 21:39:12.922813 3538281 caffe.cpp:313] Batch 4, loss = 0.967278
I0129 21:39:13.531843 3538281 caffe.cpp:313] Batch 5, accuracy/top-1 = 0.85
I0129 21:39:13.531872 3538281 caffe.cpp:313] Batch 5, accuracy/top-5 = 0.9
I0129 21:39:13.531875 3538281 caffe.cpp:313] Batch 5, loss = 0.546317
I0129 21:39:14.142321 3538281 caffe.cpp:313] Batch 6, accuracy/top-1 = 0.7
I0129 21:39:14.142349 3538281 caffe.cpp:313] Batch 6, accuracy/top-5 = 0.95
I0129 21:39:14.142364 3538281 caffe.cpp:313] Batch 6, loss = 0.853022
I0129 21:39:14.753533 3538281 caffe.cpp:313] Batch 7, accuracy/top-1 = 0.75
I0129 21:39:14.753558 3538281 caffe.cpp:313] Batch 7, accuracy/top-5 = 0.95
I0129 21:39:14.753574 3538281 caffe.cpp:313] Batch 7, loss = 0.652579
I0129 21:39:15.365084 3538281 caffe.cpp:313] Batch 8, accuracy/top-1 = 0.65
I0129 21:39:15.365109 3538281 caffe.cpp:313] Batch 8, accuracy/top-5 = 1
I0129 21:39:15.365125 3538281 caffe.cpp:313] Batch 8, loss = 0.902966
